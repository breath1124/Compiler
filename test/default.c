int main()
{
    int i = 0;
    int n = 5;
    switch(n) {
        case 0: print i+0;
        case 1: print i+1;
        case 2: print i+2;
        case 3: print i+3;
        case 5: print i+5;
        case 6: print i+6;
        default: print n*n;        
    }
}