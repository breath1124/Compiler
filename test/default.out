24 19 0 5 25 13 0 0 12 13 0 1 1 0 5 12 13 0 1 1 11 13 0 1 1 11 6 17 45 13 0 1 1 11 13 0 1 1 11 3 22 15 -1 16 52 13 0 1 1 11 18 59 13 11 22 15 -1 16 69 13 0 1 1 11 0 1 6 17 79 13 11 0 1 1 22 15 -1 16 89 13 0 1 1 11 0 2 6 17 99 13 11 0 2 1 22 15 -1 16 109 13 0 1 1 11 0 3 6 17 119 13 11 0 3 1 22 15 -1 16 129 13 0 1 1 11 0 5 6 17 139 13 11 0 5 1 22 15 -1 16 149 13 0 1 1 11 0 6 6 17 157 13 11 0 6 1 22 21 2 21 1